From 1981a8cb07e95e7b9de6705f78c94b783b126744 Mon Sep 17 00:00:00 2001
From: Jett Jackson <jacksonjett5@gmail.com>
Date: Wed, 28 Dec 2016 13:59:39 +0800
Subject: [PATCH 124/214] Added sticky footer using Flexbox

Uses vendor prefixes for 97.32% support ([Source](http://caniuse.com/#feat=flexbox)).

[Source for CSS rules](https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/).

Change-Id: I24e1104e1e9653621ec72df984dc8746bc1c65bf
---
 style.scss | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/style.scss b/style.scss
index 2842cf7..c3d1823 100644
--- a/style.scss
+++ b/style.scss
@@ -21,6 +21,14 @@ body {
 	background: $white;
   font: 18px/1.4 $helvetica;
   color: $darkGray;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  min-height: 100vh;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
 }
 
 .container {
@@ -241,6 +249,12 @@ nav {
 // .main
 //
 
+#main {
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+}
+
 .posts > .post {
   padding-bottom: 2em;
   border-bottom: 1px solid $lightGray;
-- 
2.17.1

